features/car/oneway.feature

```diff
@@ -81,3 +81,12 @@ Feature: Car - Oneway streets
         When I route I should get
             | from | to | route    |
             | a    | c  | ab,bc,bc |
+
+
+    # Reversible oneways (low frequency) vs alternating oneways (high frequency).
+    # See: https://github.com/Project-OSRM/osrm-backend/issues/2837
+    Scenario: Car - Route over alternating but not reversible oneways
+        Then routability should be
+            | highway | oneway      | forw | backw |
+            | primary | reversible  |      |       |
+            | primary | alternating | x    | x     |
```

Comment:
can this tag occur with some other items that `imply` oneway? E.g. what about `motorway` or `roundabouts`?

Not that I would think them reasonable, but I would hate to fail horribly if they should occur in the *wild*